metron-platform/metron-elasticsearch/metron-elasticsearch-common/pom.xml

```diff
@@ -26,7 +26,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <guava_version>${global_hbase_guava_version}</guava_version>
```

Comment:
Is this safe to change? ie do we have an hbase enrichment running to verify the classpath doesn't have problems?

Reply:
It looks like I don't need that change for this PR, so I reverted it. I may need to revisit this in #1454.